utilities.py

```diff
@@ -1,7 +1,7 @@
 import pandas as pd
 import cx_Oracle as orcl
 
-result_columns = ['SITE', 'CLIENT_VERSION', 'VERSION_DATE', 'DB_HOST_NAME', 'ORACLE_VERSION','ORACLE_EDITION', 'INSTANCE_STATUS', 'RESTRICTED_LOGIN', 'DB_STATUS','DB_FREE_SPACE_GB']
+result_columns = ['SITE', 'CLIENT_VERSION', 'VERSION_DATE', 'DB_HOST_NAME', 'ORACLE_VERSION','ORACLE_EDITION', 'INSTANCE_STATUS', 'RESTRICTED_LOGIN', 'DB_STATUS', 'DB_FREE_SPACE_GB', 'DB_USED_SPACE_GB']

```

Comment:
pep8

please break down each val to different line
so next time u add line it will be short diff